### PR TITLE
Fix/menu

### PR DESCRIPTION
**Pull request by ca3tiel**
make telegram link in Apprepository
remove some menu item and add Privacy Policy and Terms of Service

### CODE REVIEW

File: app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package io.nekohasekai.sagernet.vpn.nav
+
+import android.os.Bundle
+import android.widget.ImageView
+import io.nekohasekai.sagernet.R
+import io.nekohasekai.sagernet.vpn.BaseThemeActivity
+
+class TermsofServiceActivity : BaseThemeActivity() {
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt]: Make it uppser case:
```
TermsofServiceActivity => TermsOfServiceActivity
```

File: app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_termsof_service)
+
+        val ivTermsofServiceIconAngle: ImageView = findViewById(R.id.ivTermsofServiceIconAngle)
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt]: Same as https://github.com/ca3tiel/NekoBoxForAndroid/pull/17/files#r1590300729

File: app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt (hunk 1 above)
Comment (ca3tiel) [reply app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt]: fixed

File: app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt (hunk 2 above)
Comment (ca3tiel) [reply app/src/main/java/io/nekohasekai/sagernet/vpn/nav/TermsofServiceActivity.kt]: fixed

**Review by kamrava — APPROVED** [review]

**Pull request merged**